vast/vast.cpp

```diff
@@ -49,7 +49,12 @@ int main(int argc, char** argv) {
   cfg.set("logger.console", caf::atom("COLORED"));
   caf::actor_system sys{cfg};
   default_application app;
-  app.name("vast");
+  app.root.description = "manage a VAST topology";
+  app.root.name = argv[0];
+  // Skip any path in the application name.
```

Comment:
This code simply takes the application name from `argv[0]` instead of hardcoding it. However, `argv[0]` can contain a path (depending on how the user called it).

Reply:
Ah, that makes sense.